staging/src/k8s.io/code-generator/_examples/apiserver/openapi/openapi_generated.go

```diff
@@ -0,0 +1,2006 @@
+// +build !ignore_autogenerated
```

Comment:
we don't check-in this file for built-in APIs: https://github.com/kubernetes/kubernetes/blob/9027738dd5b1a4a6d5a597d9487fdadad0e7ff71/.gitignore#L113

let's ignore this one to be consistent

Reply:
I see. it's under `k8s.io/code-generator/_examples`, not `k8s.io/sample-apiserver`

Reply:
Actually for this one (and the sample-apiserver) I want to see them checked in order to track changes due to tooling. We don't check the top-level one in because API changes easily conflict. This is not the case here.